pkg/cmd/server/origin/master_config.go

```diff
@@ -309,6 +285,200 @@ func BuildMasterConfig(options configapi.MasterConfig) (*MasterConfig, error) {
 	return config, nil
 }
 
+var (
+	openshiftAdmissionControlPlugins = []string{
+		"ProjectRequestLimit",
+		"OriginNamespaceLifecycle",
+		"PodNodeConstraints",
+		"JenkinsBootstrapper",
+		"BuildByStrategy",
+		imageadmission.PluginName,
+		quotaadmission.PluginName,
+	}
+
+	kubeAdmissionPlugins = []string{
+		"RunOnceDuration",
+		lifecycle.PluginName,
+		"PodNodeConstraints",
+		"OriginPodNodeEnvironment",
+		overrideapi.PluginName,
+		serviceadmit.ExternalIPPluginName,
+		"LimitRanger",
+		"ServiceAccount",
+		"SecurityContextConstraint",
+		"BuildDefaults",
+		"BuildOverrides",
+		"AlwaysPullImages",
+		"LimitPodHardAntiAffinityTopology",
+		"SCCExecRestrictions",
+		"ResourceQuota",
+	}
+
+	combinedAdmissionControlPlugins = []string{
```

Comment:
Besides the unit test to validate these, a comment would be helpful to let anyone that's editing these chains know that they need to be kept in sync.